src/themes/lightTheme.ts

```diff
@@ -165,6 +165,7 @@ const lightTheme = createMuiTheme({
     },
     MuiFormLabel: {
       asterisk: {
+        fontWeight: 'bold',
         color: Color.ErrorMain,
       },
```

Comment:
For å flytte asterisk først i label:
```suggestion
        float: 'left',
        marginRight: '0.25rem',
        marginLeft: '-4px',
      },
```